src/modules/terms/dto/create-facility-term-response.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { RESPONSE } from '@ukef/constants';
+
+export class CreateDealResponse {
+  @ApiProperty({
+    example: RESPONSE.FACILITY_TERM_CREATED,
+  })
+  message: string;
```

Comment:
Maybe we could make it clearer (in the swagger specification and in the type system) that this value will always be `Facility Term created`

For the type system, we could do
```
message = 'FacilityTermCreated' as const;
```
and for the swagger specification I think we could document it as an enum with only 1 possible value

Reply:
I created Enum for response and return it's value. Also it's used in ApiProperty

Tried to add `as const` in couple places, but **it didn't worked**!